### PR TITLE
Marked as "slow" to improve meteor self-test default run reliability.

### PR DESCRIPTION
**Pull request by hwillson**
Pull request to help address #3569.

### CODE REVIEW

File: tools/tests/colon-converter-tests.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ if (process.platform !== "win32") {
     });
   });
 
-  selftest.define("can't build local packages with colons", function () {
```
Comment (glasser) [inline tools/tests/colon-converter-tests.js]: I'm really surprised to hear that this test is flaky; it doesn't seem to communicate with any external servers. What sort of flakiness were you seeing?

File: docs/client/full-api/tableOfContents.js [hunk 2]
```diff
@@ -317,9 +317,7 @@ var toc = [
     "markdown",
     "oauth-encryption",
```
Comment (glasser) [inline docs/client/full-api/tableOfContents.js]: This commit is from another PR

File: tools/tests/colon-converter-tests.js [hunk 3]
```diff
@@ -127,7 +127,7 @@ if (process.platform !== "win32") {
 // Tests step 3: check if old packages are converted properly to have no weird
 // paths for Windows
 
-selftest.define("package with colons is converted on Windows", function () {
```
Comment (glasser) [inline tools/tests/colon-converter-tests.js]: same with this one

File: tools/tests/command-line.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ var Sandbox = selftest.Sandbox;
 var archinfo = require('../archinfo.js');
 var release = require('../release.js');
 
-selftest.define("argument parsing", function () {
```
Comment (glasser) [inline tools/tests/command-line.js]: same with this one

**Comment by glasser** [comment]
Thanks for doing this.  I'm surprised which tests you found to be flaky, though.  Usually the flakiness comes from talking to accounts, package server, etc...

**Comment by hwillson** [comment]
Hi @glasser - sorry about the pull request pollution. I'm closing this request and will create a new with updated details. Thanks!

**Pull request closed without merging**